app/views/CalculationLPP2View.scala.html

```diff
@@ -87,13 +87,12 @@ <h1 class="govuk-heading-l govuk-!-margin-bottom-5">@messages("calculation.headi
 
   </dl>
 
-  @if(isTTPActive) {
-    @insetText(InsetText(
-      id = Some("ttp-inset-text"),
-      content = HtmlContent(getMessage("calculation.ttp.insetText"))))
-  }
-
   @if(isEstimate) {
+    @if(isTTPActive) {
+        @insetText(InsetText(
+            id = Some("ttp-inset-text"),
```

Comment:
Just thinking if you want to add a class to this so you can check if _any_ TTP content exists?

Reply:
Inset text already has a class by default "govuk-inset-text" but I could change the id to be "ttp-content", then add that to the other tpp line below?

Reply:
My thoughts were to have 'ttp-info' as a class on both the inset text and the TTP text below. That means in the test you can check if _all_ of the TTP content on the page exists or not e.g.,
parsedBody.select(".whatever-class").isEmpty shouldBe true